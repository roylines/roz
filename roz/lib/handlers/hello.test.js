const {handle} = require('./hello');
const {send} = require('../telegram');

jest.mock('../telegram');

describe('hello handler', () => {
  it('should say hello back', async () => {
    await handle();
    expect(send.mock.calls).toMatchSnapshot();
  });
});
