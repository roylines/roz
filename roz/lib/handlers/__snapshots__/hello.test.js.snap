// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello handler should say hello back 1`] = `
Array [
  Array [
    Array [
      Array [
        "I'm watching you...",
        "Always watching...",
        "Always!",
      ],
    ],
  ],
]
`;
